src/main/java/seedu/address/model/EventStorage.java

```diff
@@ -84,7 +85,13 @@ public void setEvent(Event target, Event editedEvent) {
 
         events.setEvent(target, editedEvent);
     }
-
+    /**
+     * Returns the event in the event storage with the {@code eventName}.
+     */
+    @Override
+    public Event getEvent(EventName eventName) {
+        return events.get(eventName);
```

Comment:
since you used a defensive programming strategy in `UniqueEventList::get(Eventname)` i feel like you can do the same here (include the `requireNonNull` check)

Reply:
i know it's redundant since `events.get` also does that check but redundancy is good and fun and epic. just in case you wanna add more stuff to this method (e.g. for logging)

Reply:
thats valid bruh